includes/admin.form.inc

```diff
@@ -101,7 +101,7 @@ function islandora_audio_admin_form_lame_ajax_callback(array $form, array $form_
  */
 function islandora_audio_vbr_quality_validate(array $element, array &$form_state) {
   if (!empty($element['#value'])) {
-    if ($element['#value'] < 0 || $element['#value'] > 9.99) {
+    if ($element['#value'] < 0 || $element['#value'] > 9.999) {
```

Comment:
@DiegoPino I'm validating it through Drupal's element_validate_number first. It seems to catch any non-numbers and cause a form error. Do you still suggest casting to a number? (It does sound like a good idea but I'm not sure it's necessary.)

And nice style catch on the pass/catch. I'll swap that.